README.md

```diff
@@ -3,21 +3,34 @@
 
 [![Visual Studio Marketplace](https://vsmarketplacebadge.apphb.com/version/JFrog.jfrog-vscode-extension.svg)](https://marketplace.visualstudio.com/items?itemName=JFrog.jfrog-vscode-extension)
 
+# Table of Contents
+
+- [General](#general)
+- [Viewing Dependencies Information](#viewing-dependencies-information)
+- [Configuring JFrog Xray](#configuring-jfrog-xray)
+  - [Proxy configuration](#proxy-configuration)
+    - [Proxy Authorization](#proxy-authorization)
+- [Npm Projects](#npm-projects)
+- [Go Projects](#go-projects)
+- [Pypi Projects](#pypi-projects)
+- [Troubleshooting](#troubleshooting)
+- [License](#license)
+- [Building and Testing the Sources](#building-and-testing-the-sources)
+- [Code contributions](#code-contributions)
+  - [Guidelines](#guidelines)
+
 ## General
 
 The cost of remediating a vulnerability is akin to the cost of fixing a bug.
 The earlier you remediate a vulnerability in the release cycle, the lower the cost.
 JFrog Xray is instrumental in flagging components when vulnerabilities are discovered in production systems at runtime,
 or even sooner, during the development.
 
-The JFrog VS Code Extension adds JFrog Xray scanning of npm project dependencies to your VS Code IDE.
+The JFrog VS Code Extension adds JFrog Xray scanning of npm, Go and Python project dependencies to your VS Code IDE.
 It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their VS Code IDE.
 With this information, a developer can make an informed decision on whether to use a component or not before it gets entrenched into the organization’s product.
 
-## Using JFrog Visual Studio Code Extension
-
-Connect to JFrog Xray by clicking on the green Connect ![Connect](resources/readme/connect.png) button:
-![Connect](resources/readme/gifs/connect.gif)
+## Viewing Dependencies Information
 
 View the project's dependency tree:
```

Comment:
View the project's dependency tree: -->
View the dependencies used by the project in a tree, where the direct dependencies are at the top: